ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -89,9 +90,29 @@ const INITIAL_STATE: State = {
   pan: none,
   expirationDate: none,
   securityCode: none,
-  holder: none
+  holder: none,
+  isValidHolder: undefined,
+  isValidPan: undefined,
+  isValidExpirationDate: undefined,
+  isValidSecurityCode: undefined
 };
 
+function isPanValid(pan: string): boolean {
+  return CreditCardPan.is(pan);
+}
+
+function isExpirationDateValid(expirationDate: string): boolean {
```

Comment:
```suggestion
private isExpirationDateValid = (expirationDate: string) => {
```

Reply:
Here I supposed to consider the function as an alias and use the existing validator as for the previously commented objects.

Reply:
Here I supposed to consider the function as an alias and use the existing validator as for the previously commented objects.